node/src/admin_client.ts

```diff
@@ -22,6 +22,10 @@ export class AdminClient {
     );
   }
 
+  getHost(): string {
+    return this.host;
+  }
+
```

Comment:
I added a getter here because I need the host information in RequestCoordinator.